2022-01-31 00:55:43,378  -  An exception occurred in the server.
Traceback (most recent call last):
  File "/mnt/c/Users/guilh/code/voca/backend/src/controller/language_controller.py", line 170, in res_calculate_score
    translated_word: str = self.translation_model.translate(
  File "/mnt/c/Users/guilh/code/voca/backend/src/model/translation_model.py", line 36, in translate
    translation: str = self.translator.translate(word, dest=to_language).text
  File "/mnt/c/Users/guilh/code/voca/backend/venv/lib/python3.8/site-packages/googletrans/client.py", line 219, in translate
    parsed = json.loads(data[0][2])
  File "/usr/lib/python3.8/json/__init__.py", line 341, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
